packages/blockchain-wallet-v4-frontend/src/modals/BuySell/EnterAmount/Checkout/validation.tsx

```diff
@@ -119,10 +123,13 @@ export const getMaxMin = (
   sddLimit = LIMIT,
   limits?: SwapUserLimitsType
 ): { CRYPTO: string; FIAT: string } => {
-  let quote: BSQuoteType | { quote: SwapQuoteType; rate: number }
+  let quote:
+    | BSQuoteType
+    | { quote: SwapQuoteType; rate: number }
+    | { pair: string; quote: BuyQuoteType; rate: number }
   switch (orderType as OrderType) {
     case OrderType.BUY:
-      quote = QUOTE as BSQuoteType
+      quote = QUOTE as BSQuoteType | { pair: string; quote: BuyQuoteType; rate: number }
```

Comment:
This `{ pair: string; quote: BuyQuoteType; rate: number }` doesn't have its own type?

Reply:
Good point, i'm rewriting this code in quite a lot of places. Going to add this type 


`type BuyQuoteStateType = { pair: string; quote: BuyQuoteType; rate: number }`